graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/NodeHelper.java

```diff
@@ -163,10 +177,18 @@ public static JCRNodeWrapper getNodeInLanguage(JCRNodeWrapper node, String langu
             return node;
         }
         Locale locale = LanguageCodeConverters.languageCodeToLocale(language);
-        JCRSessionWrapper session = JCRSessionFactory.getInstance().getCurrentUserSession(workspace, locale);
+        JCRSessionWrapper session = JCRSessionFactory.getInstance().getCurrentUserSession(workspace, locale,
+                useFallbackLanguage  ? getFallbackLanguage(node): null);
         return session.getNodeByIdentifier(node.getIdentifier());
     }
 
+    private static Locale getFallbackLanguage(JCRNodeWrapper node) throws RepositoryException {
+        if (node.getResolveSite().isMixLanguagesActive()) {
+            return LanguageCodeConverters.languageCodeToLocale(node.getResolveSite().getDefaultLanguage());
```

Comment:
Double call to `node.getResolveSite()` could be avoided by storing first call in a variable.